encoder/vaapiencoder_h264.cpp

```diff
@@ -1215,29 +1214,29 @@ bool VaapiEncoderH264::ensurePictureHeader(const PicturePtr& picture, const VAEn
     return true;
 }
 
-static void fillReferenceList(VAEncSliceParameterBufferH264* slice, const vector<ReferencePtr>& refList, uint32_t index)
+bool VaapiEncoderH264::fillReferenceList(VAEncSliceParameterBufferH264* slice) const
```

Comment:
why we need change this code. Original one seems better, it did not duplicate business logical